### PR TITLE
Simple ekf test

### PR DESCRIPTION
**Pull request by PNieck**
## Description

Adds script to generate ekf test scenario with constant data

## Motivation and Context

EKF functional tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [x] Tested by hand on: (list targets here). `host-generic-pilot`

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing linter checks and tests passed.
- [x] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [x] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
https://github.com/phoenix-pilot/phoenix-pilot-core/pull/270
- [ ] I will merge this PR by myself when appropriate.

### CODE REVIEW

**Comment by niewim19** [comment]
Apart from changes in `models` this PR adds scripts for data generation. Currently it is generating data in some arbitrary way. Please add adequate comments which describe what each script does. Alternatively a documentation file may be created, which describes these scripts in details. Maybe a `README.md` file?

**Comment by PNieck** [comment]
> Apart from changes in `models` this PR adds scripts for data generation. Currently it is generating data in some arbitrary way. Please add adequate comments which describe what each script does. Alternatively a documentation file may be created, which describes these scripts in details. Maybe a `README.md` file?

Done

**Review by niewim19 — APPROVED** [review]

File: libs/parser/parser.c [hunk 1]
```diff
@@ -35,7 +35,7 @@
 #define HEADER_PATTERN        "^[[:space:]]*@" WORD "[[:space:]]*(#.*)?$"
 #define HEADER_SUBEXPRESSIONS 2
 
-#define FIELD_VALUE          "([A-Za-z0-9_,.+-]+)"
+#define FIELD_VALUE          "([A-Za-z0-9_,.+-/]+)"
```
Comment (kemonats) [inline libs/parser/parser.c:38]: You merged this PR but a litle note: `+-/` means chars from `+` to `/` that is `+,-./` ( you repeated `,.` ) .
If you want only hyphen character put it as first or lasta character in character class: `"([A-Za-z0-9_,.+/-]+)"`

Comment (PNieck) [reply libs/parser/parser.c:38]: > You merged this PR but a litle note: `+-/` means chars from `+` to `/` that is `+,-./` ( you repeated `,.` ) . If you want only hyphen character put it as first or lasta character in character class: `"([A-Za-z0-9_,.+/-]+)"`

Fix in separate PR https://github.com/phoenix-pilot/phoenix-pilot-core/pull/279

**Pull request merged**